pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -152,6 +160,12 @@ resources:
     expression: "0 0 * * *"
     fire_immediately: true
 
+- name: census-rm-kubernetes-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-kubernetes.git
+    private_key: ((github.service_account_private_key))
```

Comment:
I don't think this is needed, we've already got a resource for the optional directory which is all this is being used for

Reply:
Removed this resource, now using `census-rm-kubernetes-optional-repo` instead